src/connector/src/connector_common/common.rs

```diff
@@ -630,6 +630,26 @@ impl NatsCommon {
         stream: String,
         split_id: String,
         start_sequence: NatsOffset,
+        durable_name: Option<String>,
+        description: Option<String>,
+        ack_policy: Option<String>,
+        ack_wait: Option<Duration>,
+        max_deliver: Option<i64>,
+        filter_subject: Option<String>,
+        filter_subjects: Option<Vec<String>>,
+        replay_policy: Option<String>,
+        rate_limit: Option<u64>,
+        sample_frequency: Option<u8>,
+        max_waiting: Option<i64>,
+        max_ack_pending: Option<i64>,
+        _idle_heartbeat: Option<Duration>,
```

Comment:
Is this deprecated? May just remove it.

Reply:
Well the behavior of the [missed idle heartbeat](https://docs.rs/async-nats/latest/async_nats/jetstream/consumer/struct.Config.html#structfield.idle_heartbeat) is a bit cryptic with the ‘async_nats’ crate and is a relatively important parameter user because it translates into error messages sent by the stream in the form of “missed idle heartbeat”, which can cause pipelines to fail. Not sure how to handle this one tbh

Reply:
Removed from the configuration, since it doesn't exist in the pull consumer config: https://docs.rs/async-nats/latest/async_nats/jetstream/consumer/pull/struct.Config.html

Reply:
Removed from the configuration, since it doesn't exist in the pull consumer config: https://docs.rs/async-nats/latest/async_nats/jetstream/consumer/pull/struct.Config.html